proto/servicemanager/v4/servicemanager.templ

```diff
@@ -0,0 +1,165 @@
+#include <aos/common/types.hpp>
+#include <aos/common/ocispec.hpp>
+
+// SMIncomingMessages
+
+// CheckNodeConfig
+servicemanager.v4.CheckUnitConfig.node_config max_length:1024
+servicemanager.v4.CheckUnitConfig.version     max_length:${aos::oci::cVersionLen}
+
+// SetUnitConfig
+servicemanager.v4.SetUnitConfig.node_config max_length:1024
+servicemanager.v4.SetUnitConfig.version     max_length:${aos::oci::cVersionLen}
+
+// UpdateNetworks
+servicemanager.v4.UpdateNetworks.networks max_count:4
+
+// RunInstances
+servicemanager.v4.RunInstances.services  max_count:${aos::cMaxNumServices}
+servicemanager.v4.RunInstances.layers    max_count:${aos::cMaxNumLayers}
+servicemanager.v4.RunInstances.instances max_count:${aos::cMaxNumInstances}
+
+// ServiceInfo
+servicemanager.v4.ServiceInfo.service_id  max_length:${aos::cServiceIDLen}
+servicemanager.v4.ServiceInfo.provider_id max_length:${aos::cProviderIDLen}
+servicemanager.v4.SetUnitConfig.version   max_length:${aos::oci::cVersionLen}
+servicemanager.v4.ServiceInfo.url         max_length:${aos::cURLLen}
+servicemanager.v4.ServiceInfo.sha256      max_size:${aos::cSHA256Size}
+
+// LayerInfo
+servicemanager.v4.LayerInfo.layer_id max_length:${aos::cLayerIDLen}
+servicemanager.v4.LayerInfo.digest   max_length:${aos::cLayerDigestLen}
+servicemanager.v4.LayerInfo.url      max_length:${aos::cURLLen}
+servicemanager.v4.LayerInfo.sha256   max_size:${aos::cSHA256Size}
+
+// FirewallRule
+servicemanager.v4.FirewallRule.dst_ip   max_length:16
```

Comment:
We need to consider ipv6  as well. Probably it would be better to set a max_length to 40 chars

Reply:
Will create a JIRA task to make all these magic number constants

Reply:
https://jiraeu.epam.com/browse/EPMPAOS-5250